packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -741,7 +741,7 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
       this.callClient,
       this.callAgent,
       this.deviceManager,
-      undefined,
+      this.call,
```

Comment:
do we need to call `createHandlers` in a leave call scenario? Can we just leave them as is and delete this function call?

Reply:
Let me check who wrote this line of code and verify

Reply:
Prabhjot wrote this in this PR https://github.com/Azure/communication-ui-library/pull/3352/files we should probably keep this for now